manifest.konnector

```diff
@@ -1,14 +1,14 @@
 {
   "version": "1.0.0",
-  "name": "Connector template",
+  "name": "Connector deliveroo",
   "type": "konnector",
   "language": "node",
-  "icon": "icon.png",
-  "slug": "template",
-  "source": "git://github.com/konnectors/cozy-konnector-template.git",
+  "icon": "icon.jpg",
```

Comment:
The store can't display jpg images. I must be png or svg to my knowledge.

Reply:
I wish I could use a SVG icon if I can find it, but I can't. AFAIK jpg works as well as png. What do you want me to do?

Reply:
jpg works, let's merge and ask our amazing designer for svg as later optimisations.